docs/articles/documentation/recipes/using-testcafe-with-gulp.md

```diff
@@ -53,7 +53,7 @@ gulp run-testcafe-tests
 Or create and run a dependent task.
 
 ```js
-gulp.task('build', ['run-testcafe-tests'], () => {
+gulp.task('publish', ['run-testcafe-tests'], () => {
```

Comment:
Maybe just `gulp.task('run-testcafe-tests')` ?
I suppose gulp users know what they should do with it